examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>Time is always precious :)</p><p> </p><blockquote>Comparing to &lt;&gt;, we can see, that <code>microbatch</code> allows us to leverage simple trade-off between bigger batch size (thus model performance) and model training time. If the data in batch can fit memory constraints, there is no reason to use <code>microbatch</code> due to inherently slower processing of batches.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>